TodoApp.ConsoleApp/UI/TaskOperations.cs

```diff
@@ -79,15 +77,15 @@ static void ManipulateTask()
                     break;
 
                 case 2:
-                    XMLTaskReader reader = new XMLTaskReader();
-                    List<Task> tasks = reader.ReadTasks("../../tasks.xml");
-                    int n = int.Parse(UserInput.ReadText("Select the number of the task, you want to delete: ", true));
-                    XMLTaskWriter writer = new XMLTaskWriter();
-                    writer.Delete(tasks[n - 1]);
-                    UserInput.ReadText("Delete completed!");
+                    IEnumerable<Task> tasks = service.GetAll();
+                    string id = UserInput.ReadText("Select the number of the task, you want to delete: ", true);
+                    if (Guid.TryParse(id,out newGuid))
```

Comment:
User trying to enter a GUID is not a good practice. You can use:
```
int index = UserInput.ReadInt( ... );
service.DeleteByIndex(index);
```
You must add two new methods.
- For TodoService:
```
bool DeleteByIndex(int index)
{
  var tasks = this.GetAll();
  var taskId = tasks[index].Id;
  return this.Delete(taskId);
}
```

- For UserInput. Implementation is similar to the other Read methods; min/max is the minimum and maximum possible value:
```
int ReadInt(string question, int min, int max, bool required = true, int defaultValue = 0);
```